modules/andes-core/broker/src/main/java/org/wso2/andes/store/cassandra/CQLBasedAndesContextStoreImpl.java

```diff
@@ -24,22 +24,14 @@
 import com.datastax.driver.core.querybuilder.QueryBuilder;
 import com.datastax.driver.core.schemabuilder.SchemaBuilder;
 import org.wso2.andes.configuration.util.ConfigurationProperties;
-import org.wso2.andes.kernel.AndesBinding;
-import org.wso2.andes.kernel.AndesContextStore;
-import org.wso2.andes.kernel.AndesException;
-import org.wso2.andes.kernel.AndesExchange;
-import org.wso2.andes.kernel.AndesQueue;
-import org.wso2.andes.kernel.DurableStoreConnection;
-
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-
-import static com.datastax.driver.core.querybuilder.QueryBuilder.decr;
-import static com.datastax.driver.core.querybuilder.QueryBuilder.eq;
-import static com.datastax.driver.core.querybuilder.QueryBuilder.incr;
-import static com.datastax.driver.core.querybuilder.QueryBuilder.set;
+import org.wso2.andes.kernel.*;
```

Comment:
could you remove util.*;
and kernel.* too.
import explicitly ( - that adheres to our coding standard)